libpod/runtime_img.go

```diff
@@ -1262,3 +1266,219 @@ func getPolicyContext(ctx *types.SystemContext) (*signature.PolicyContext, error
 	}
 	return policyContext, nil
 }
+
+// sizer knows its size.
+type sizer interface {
+	Size() (int64, error)
+}
+
+func imageSize(img types.ImageSource) *uint64 {
+	if s, ok := img.(sizer); ok {
+		if sum, err := s.Size(); err == nil {
+			usum := uint64(sum)
+			return &usum
+		}
+	}
+	return nil
+}
+
+func reposToMap(repotags []string) map[string]string {
+	// map format is repo -> tag
+	repos := make(map[string]string)
+	for _, repo := range repotags {
+		var repository, tag string
+		if len(repo) > 0 {
+			li := strings.LastIndex(repo, ":")
+			repository = repo[0:li]
+			tag = repo[li+1:]
+		}
+		repos[repository] = tag
+	}
+	if len(repos) == 0 {
+		repos["<none>"] = "<none"
+	}
+	return repos
+}
+
+// GetImagesForImages gets the images for podman images and returns them as
```

Comment:
Nit: "gets the images" -> "gets all images"

Reply:
i am specifically avoiding that term until we have a notion of "all" images.